platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -56,6 +65,25 @@ SECTIONS
         __vectors_end__ = .;
     } > FLASH
 
+#ifndef TFM_MULTI_CORE_TOPOLOGY
+    /*
+     * Place the CMSE Veneers (containing the SG instruction) after the code, in a
+     * separate 32 bytes aligned region so that the SAU can programmed to just set
+     * this region as Non-Secure Callable.
+     */
+    .gnu.sgstubs : ALIGN(32)
+    {
+        SG_ALIGN;
+        Load$$LR$$LR_VENEER$$Base = ABSOLUTE(.);
+        *(.gnu.sgstubs*)
+    } > VENEERS AT> VENEERS
+    SG_END_ALIGN;
+    Load$$LR$$LR_VENEER$$Limit = .;
```

Comment:
This value is [written](https://github.com/NordicPlayground/trusted-firmware-m/blob/cf25ed7a1f1ac14a1f7244877c4485311cb76c91/platform/ext/target/nordic_nrf/nrf5340/target_cfg.c#L51) directly to `memory_regions.veneer_limit`, so it should be probably `. -1` here (or this initialization of `memory_regions` needs to be corrected). Otherwise, configuration of the NSC region ends up in HardFault.

Reply:
Thanks, fixed